css-ui-4/Overview.bs

```diff
@@ -2199,6 +2203,42 @@ that are typical of the element whose appearance it acquires.
 	is also not changed by the 'appearance' property.
 </div>
 
+<h3 id="input-security">Obscuring sensitive input: the 'input-security' property</h3>
+
+<pre class=propdef>
+Name: input-security
+Value: auto | none
+Initial: ''input-security/auto''
+Applies to: elements that accept sensitive text input, such as <{input/type/password|&lt;input type=password&gt;}>
+Inherited: no
+Percentages: N/A
+Computed Value: the keword ''input-security/auto'' or the keyword ''input-security/none''
+Canonical order: per grammar
+Animation type: by computed value type
+</pre>
+
+When the purpose of a text input control is to accept sensitive input,
+such as <{input/type/password|&lt;input type=password&gt;}> in HTML,
+user agents obscure the display of the control's content by default
+in order to prevent onlookers from seeing it.
+Users may wish to temporarily disable this obscuring
+in order to confirm that they've typed their password correctly.
+The ''input-security'' property may be used by authors
+to enable or disable this obscuring.
+
+<dl dfn-type=value dfn-for=input-security>
+	<dt><dfn>none</dfn>
+	<dd>
+		The UA must not obscure the text in the control,
+		so that it can be read by the user.
+	<dt><dfn>auto</dfn>
+	<dd>
+		The UA should obscure the text in the control,
+		so that it cannot be read by the user.
+</dl>
+
+The exact mechanism by which user agents obscure the text in the control is undefined.
```

Comment:
Keeping it undefined is correct, but this could use an example showing off the dot-replacement used by current UAs.

Reply:
I've provided an example in 3f5738a.